### DISCUSSION TITLE
How to close the ice connection when iframe or video is hidden or paused

### DISCUSSION THREAD

**Discussion by krishgcek** [bluenviron/mediamtx#2712] [Questions]
### Question

I am using iframe with webrtc.I have many iframes in a page. This creates lot's of connections.How is the peer connection is closed when iframe is hidden or video is paused?

**Comment by aler9** [comment]
Hello, if you want to close the WebRTC peer connection, you have to close the iframe. My advice is to perform this with javascript, for instance by setting src to an empty string:

```js
if element is off screen {
   document.getElementById("my_iframe").src = "";
} else {
   document.getElementById("my_iframe").src = "real url";
}
```

there's no support (yet) for pausing the stream.

[a comment by an automated account is not shown]
